### DISCUSSION TITLE
Talk about UI in osu!

### DISCUSSION THREAD

**Discussion by MeuhMeuh62** [ppy/osu#27493] [Ideas]
I made on Figma a concept which change UI multiplayers rooms (here in team VS)
It is the first version :
![image](https://github.com/ppy/osu/assets/86471514/dc7bff42-d301-4da6-8bee-7d99a9d13b13)
Red team left, blue team right, chat on middle.
(Icons are picked on internet dw haha)
I add a feature a "pin message" host or referee can do, for example: `!mp pin hello!`, "hello!" will pinned.
Add the timer UI too (the yellow progress bar show time left). 
Picked map are in the center and can be changed by the host/referee
For casuals rooms and show which bm follow after, players can click on "beatmap" category and demote chat just pin message stayed
![image](https://github.com/ppy/osu/assets/86471514/a71530c4-47ea-48dd-8f60-01261a47f874)
Extra: 
Dispositions of teams :
![image](https://github.com/ppy/osu/assets/86471514/01e2d04c-343b-4dc2-b3b6-c84982d9fa7d)

It's a concept, can be edited and add features, i know is not perfect lol
I will add AFK button #27461 & more soon
↑ Up this idea to be interested

**Comment by Stoppedpuma** [comment]
Designs are already being made: https://www.figma.com/file/XJCRjfvTlVIwZBWoAz5kB2/Client%2FPlaylist-Multiplayer?type=design&node-id=0-1&mode=design&t=VY2Myd49WwAqQYsC-0

**Reply by MeuhMeuh62** [reply]
Is it an official figma for a next release?

**Reply by Stoppedpuma** [reply]
Yes these are flytes designs, they are still wip so they probably wont be seen in a release for a while.

**Reply by MeuhMeuh62** [reply]
Oh ok! The UI is pretty nice and matches with the new Lazer UI.
I saw tournament mode because I am a referee, it's really nice and dodge simples IRC chats.
I have some features to add, there exists a thread to speak about this?

**Reply by Stoppedpuma** [reply]
I would say using figma would work for suggesting changes in areas of the UI but all of these designs are being updated to new standards like auto layouts and being reorganised for simplicity since they are kind of a mess file structure wise.

It would probably be a good idea to change this thread to a discussion thread since the page will probably be moved to a new one after it's updated.

**Reply by MeuhMeuh62** [reply]
Yes i agree
For me i can't change "ideas" cathegory but let's talk about UI there!

**Comment by MeuhMeuh62** [comment]
@Stoppedpuma do you know for the new chat UI?

**Reply by Stoppedpuma** [reply]
The chat has already seen an updated design recently, you can find those chat designs [here](https://www.figma.com/file/f8b2dHp9LJCMOqYP4mdrPZ/Client%2FChat?type=design&node-id=0-1&mode=design&t=Fv1T5t9xwcRGU6At-0).

You can find most of the other designs [here](https://www.figma.com/file/VIkXMYNPMtQem2RJg9k2iQ/Master-Library?type=design&node-id=6784-325&mode=design&t=VrZ9S5DaaOOH5tMj-0). There's quite a few that aren't listed here so if you can't find something just let me know and I'll send you a link if flyte has made designs for it.
